doc/changes.txt

```diff
@@ -38,9 +38,8 @@ New features:
   @vdblink::points::AttributeSet::removeAttribute()
   AttributeSet::removeAttribute@endlink for releasing ownership of attribute
   data.
-- Added @vdblink::points::AttributeSet::Descriptor::groupIndexCollision()
-  AttributeSet::Descriptor::groupIndexCollision@endlink for detecting group
-  index collisions when attempting to merge two Descriptors.
+- Added AttributeSet::Descriptor::groupIndexCollision for detecting group index
+  collisions when attempting to merge two Descriptors.
```

Comment:
It looks like the @vdblink mechanism doesn't work for functions of a nested class, so I've removed them here.